modules/wordads/wordads.php

```diff
@@ -113,10 +113,16 @@ private function insert_adcode() {
 		add_action( 'wp_head', array( $this, 'insert_head_meta' ), 20 );
 		add_action( 'wp_head', array( $this, 'insert_head_iponweb' ), 30 );
 		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
-		add_filter( 'the_content', array( $this, 'insert_ad' ) );
-		add_filter( 'the_excerpt', array( $this, 'insert_ad' ) );
 
-		if ( $this->option( 'enable_header_ad' ) ) {
+		if ( ! apply_filters( 'wordads_content_disable', false ) ) {
```

Comment:
One more nitpick - can you please add filter headers here? Here is an example of a comment block that will be picked up by the parser: https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L1552

Reply:
Added.